### ISSUE TITLE
alt+] doesn't go to the last chat response

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#188829]
Testing #188537

I can use alt+[ and ] to navigate all chat responses... except for the very last response. alt+] does nothing when I'm on the second to last response. Suspecting an off-by-one somewhere

**Comment by rzhao271** [comment]
I seem to be unable to open the view on the latest Insiders.
I confirmed the issue exists on the previous Insiders.
